src/popup/FormattedTextComponent.tsx

```diff
@@ -19,7 +19,7 @@ export const FormattedText = (props: FormattedTextProps) => {
             (async () => {
                 const text = "text" in props ? props.text : chrome.i18n.getMessage(props.langKey);
 
-                setLabel(await formatTitleInternal(text, false, props.titleFormatting!, false));
+                setLabel(await formatTitle(text, false, props.titleFormatting!, false, Config.config!.onlyTitleCaseInEnglish));
```

Comment:
What's the reason for this change?

Reply:
There are two changes here:
1. **Replace `formatTitleInternal` → `formatTitle`**: note that `formatTitle` comes from DeArrowFormatting, not from `titles/titleFormatter.ts`. I just moved `formatTitleInternal` in DeArrowFormatting, and renamed it to `formatTitle`.
    - **As to why I changed the name**: I thought it didn't make sense to label the function "internal" when, from the PoV of DeArrowFormatting, it was the function being exposed to the outside.
      From the PoV of DeArrow, the function is used internally (which is part of the reason why [I aliased it to `formatTitleInternal` in `src/titles/titleFormatter.ts`](https://github.com/ajayyy/DeArrow/pull/242/files#diff-ee7254a39e09cbcaf11723f3c630344f85e959ed7edaade39902cee225867401R4))
2. **Added `Config.config!.onlyTitleCaseInEnglish` parameter**: DeArrowFormatting doesn't know about DeArrow's `Config` object anymore, so we have to pass it manually.